### PR TITLE
Scala js respin

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,34 +1,72 @@
+import sbt.Project.projectToRef
+
+lazy val clients = Seq(client)
```
Comment (novakov-alexey) [inline build.sbt]: I think val clients should be moved closer to its usage (i.e. before server val)

File: build.sbt [hunk 2]
```diff
@@ -1,34 +1,72 @@
...
-  libraryDependencies ++= Seq(
-    "org.webjars" % "jquery" % "2.2.0",
+    libraryDependencies ++= Seq(
+      "org.webjars" % "jquery" % "2.2.0",
+      "com.vmunier" %% "play-scalajs-scripts" % "0.4.0",
 
-    "com.datastax.cassandra" % "cassandra-driver-core" % "3.0.0-rc1" exclude("org.xerial.snappy", "snappy-java"),
-    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0", // todo remove
+      "com.datastax.cassandra" % "cassandra-driver-core" % "3.0.0-rc1" exclude("org.xerial.snappy", "snappy-java"),
+      "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0", // todo remove
```
Comment (novakov-alexey) [inline build.sbt]: we can remove it right now or it is used already?

File: build.sbt [hunk 3]
```diff
@@ -1,34 +1,72 @@
...
-    "org.scalatestplus" %% "play" % "1.4.0" % "test",
-    "org.scalamock" %% "scalamock-scalatest-support" % "3.2.2" % "test"
+lazy val client = (project in file("client"))
+  .enablePlugins(ScalaJSPlugin, ScalaJSPlay)
+  .dependsOn(sharedJs)
+  .settings(commonSettings: _*)
+  .settings(
+    persistLauncher := true,
+    persistLauncher in Test := false,
+    //    sourceMapsDirectories += sharedJs.base / "..",
```
Comment (novakov-alexey) [inline build.sbt]: can you remove commented code?

File: build.sbt [hunk 4]
```diff
@@ -1,34 +1,72 @@
...
+      "be.doeraene" %%% "scalajs-jquery" % "0.8.1",
+      "org.monifu" %%% "monifu" % "1.0"
+    )
   )
-)
+
+lazy val shared = (crossProject.crossType(CrossType.Pure) in file("shared"))
+  .settings(commonSettings: _*)
+  .jsConfigure(_ enablePlugins ScalaJSPlay)
+//  .jsSettings(sourceMapsBase := baseDirectory.value / "..")
```
Comment (novakov-alexey) [inline build.sbt]: commented code

**Comment by novakov-alexey** [comment]
Rest is fine

File: build.sbt (hunk 2 above)
Comment (DmytroOrlov) [inline build.sbt]: even heavily used

Comment (novakov-alexey) [inline build.sbt]: Ok

**Pull request merged**